### ISSUE TITLE
No QR Code generated after request

### ISSUE THREAD

**Issue by bialeta** [thunderbird/thunderbird-android#8469]
### Checklist

- [X] I have used the search function to see if someone else has already submitted the same bug report.
- [X] I will describe the problem with as much detail as possible.

### App

Thunderbird for Android

### App version

8.0

### Where did you get the app from?

Google Play

### Android version

14

### Device model

SM-G991B/DS (Galaxy S21 5G)

### Steps to reproduce

Ubuntu 24.04.1 LTS; TB 128.4.0esr (64-bit)
From desktop: Tools -> Export for mobile -> checkbox all email accounts and passwords or every single account with or without password checkbox ticked.

### Expected behavior

Show QR Code

### Actual behavior

Nothing, no QR Code generated.

### Logs

JavaScript error: resource:///modules/qrcode.mjs, line 169: uncaught exception: Cannot encode text

**Comment by kewisch** [comment]
Would you mind filing a desktop bug on this at https://bugzilla.mozilla.org ? Product Thunderbird, Component Account Manager. If you are using multibyte characters (e.g. Japanese), it might be https://bugzilla.mozilla.org/show_bug.cgi?id=1928114 which is set to be fixed in the next ESR point release. 

Desktop might also need a little more information on your setup to understand which text could not be encoded.

The meta bug for all desktop QR issues is https://bugzilla.mozilla.org/show_bug.cgi?id=1918537

**Comment by cmj2002** [comment]
@kewisch Does this fix included in version `128.4.1esr`? I just updated to `128.4.1esr`, but the issue still persists.

I originally wanted to comment under the [bugzilla bug 1928114](https://bugzilla.mozilla.org/show_bug.cgi?id=1928114) you mentioned, but I found that I don't have permission to comment on closed bugs.

I am running Thunderbird `128.4.1esr` (64bit) on Windows 11 (22631.4317), with the same error in [bugzilla bug 1928114](https://bugzilla.mozilla.org/show_bug.cgi?id=1928114):

```
Uncaught Cannot encode text [qrcode.mjs:169:9](resource:///modules/qrcode.mjs)
    Segment resource:///modules/qrcode.mjs:169
    segments resource:///modules/qrcode.mjs:818
    map self-hosted:178
    generate resource:///modules/qrcode.mjs:818
    renderQR resource:///modules/QRExport.sys.mjs:214
    getQRCodes resource:///modules/QRExport.sys.mjs:132
    initializeQRCodes chrome://messenger/content/preferences/qr-code-wizard.mjs:58
    showCodes chrome://messenger/content/preferences/qrExport.mjs:255
    addEventListeners chrome://messenger/content/preferences/qrExport.mjs:109
```

**Comment by cmj2002** [comment]
I did some tests. My accounts' `Default Identity > Your Name` in `Account Setting` are in Chinese, and if I change them to English, I can generate QR code successfully. So it seems that the character encoding issue has not been completely resolved, or the fix was not included in `128.4.1esr`?

**Comment by cketti** [comment]
You can get this information from the Bugzilla bug. So far it has only been included in Thunderbird 133.0b1 (beta). A comment will be added once it has been included in an ESR release.

**Comment by cmj2002** [comment]
I just updated to 128.4.2esr and the issue is fixed. Thank you guys!
